### DISCUSSION TITLE
Pub Sub connector With Cloud Interconnect?

### DISCUSSION THREAD

**Discussion by svetakvsundhar** [syslog-ng/syslog-ng#4808] [General]
Hi all,

I am interested in using https://github.com/syslog-ng/syslog-ng/pull/4651 (https://axoflow.com/google-pub-sub-and-openobserve-support-in-syslog-ng-version-4-5/), and want to confirm that a common use case here is to use this via a tool like [Cloud Interconnect](https://cloud.google.com/network-connectivity/docs/interconnect/concepts/overview). That is, this connector can be used in the case that `syslog-ng` is in one VPC and `Pub Sub` is in another VPC.

**Comment by bazsi** [comment]
As long as IP connectivity is there, it should work, yes.
